checkout-core/src/main/java/com/adyen/checkout/core/api/OkHttpClient.kt

```diff
@@ -49,7 +52,18 @@ internal class OkHttpClient(
                 ?.bytes()
                 ?: ByteArray(0)
         } else {
-            throw IOException(response.body?.string())
+            val errorBody = try {
```

Comment:
Were you able to make an API call that returns an error to test it? If not I can maybe suggest something

Reply:
Yes, I tested it by playing with the bin lookup call